composer.json

```diff
@@ -24,13 +24,13 @@
         "predis/predis"  : "v0.8.5",
         "guzzle/guzzle": ">=3.8@stable",
         "doctrine/common": "^2.5",
-        "talis/persona-php-client": ">=3.1"
+        "talis/persona-php-client": "^4.0.0"
```

Comment:
Is it okay to use ^4.0.0 here rather than locking onto a specific version?

I'm upgrading the persona-php-client in TARL. To do that I need the same version of the client in the echo-php-client, the babel-php-client, the critic-php-client and TARL itself. If we don't lock onto specific versions how do we ensure that i's the samwe version everywhere?